scripts/utils.py

```diff
@@ -177,3 +177,40 @@ def save_metadata(s3_client,
         finally:
             os.remove(fname)
 
+######################
+# Shapefile uploader #
+######################
+
+def upload_shapefile(s3_client,
```

Comment:
Should this be `save_shapefile()`?

Reply:
I don't think so. The `save_data` function does two things: saves the file locally and then uploads it to AWS. The `upload_shapefile` only uploads - the shapefile has already been saved as part of the product generation process.

Oh, except I've used the wrong name for this function in the worker! Will fix now.

Reply:
Makes sense. It's indeed the import within the worker that prompted this comment.